pyproject.toml

```diff
@@ -181,7 +181,7 @@ test-integration = 'sh scripts/test/integration.sh'
 test-kubernetes = "sh scripts/test/integration-kubernetes.sh"
 test-kubernetes-setup = "sh scripts/test/kubernetes-setup.sh"
 test-integration-dbt-1-5-4 = 'sh scripts/test/integration-dbt-1-5-4.sh'
-test-integration-dbt-async = 'sh scripts/test/integration-dbt-async.sh  {matrix:dbt}'
+test-integration-dbt-async = 'bash scripts/test/integration-dbt-async.sh  {matrix:dbt}'
```

Comment:
`sh` does not support the conditional logic with syntax `[[` and `]]` used to test dbt version 1.6 & 1.9  and hence using bash which supports that.